CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 * Bumps Instabug Android SDK to v11.11.0.
+* Bumps Instabug iOS SDK to v11.10.0.
```

Comment:
We should change "Bumps" here and for Android to "Bump" following the new guidelines.

Reply:
The new changelog guidelines will be applied in a separate PR that follows this.